README.md

```diff
@@ -55,18 +55,9 @@ and so on.
 
 `directory/*/*.csv` matches only `directory/[subdirectory]/[filename].csv`. With a [recursive glob pattern](https://docs.python.org/3/library/glob.html#glob.glob), `directory/**/*.csv` will additionally match:
 
-* `directory/[filename].csv` (no subdirectory)
-* `directory/[subdirectory1]/[subdirectory2]/[filename].csv` (sub-subdirectory)
+- `directory/[filename].csv` (no subdirectory)
+- `directory/[subdirectory1]/[subdirectory2]/[filename].csv` (sub-subdirectory)
 
 and so on, for as many levels deep as exist in the directory tree.
 
 </details>
-
-
-
-## MongoDB
```

Comment:
Is it worth noting something like "This used to require Mongo but doesn't anymore"?

Reply:
The app still uses mongo, but we don't write to it directly anymore